packages/ansible-language-server/src/errors.ts

```diff
@@ -129,6 +129,13 @@ ERRORS.addError(
     "Cloudflare rejected the request. Please contact your administrator.",
   ),
 );
+ERRORS.addError(
+  400,
+  new Error(
+    "error__wca_hap_filter_rejection",
+    "IBM watsonx Code Assistant Hate, Abuse, and Profanity filter rejected your request. Please try again.",
```

Comment:
If the input is unchanged, re-submitting the playbook generation/explanation request will be rejected again...  I guess we need to tell the user something like "Please eliminate any doubtful phrases and try again".

Reply:
@TamiTakamiya Agreed.. I had thought that too. I'll update.